awx/main/conf.py

```diff
@@ -323,9 +323,9 @@ def _load_default_license_from_file():
 register(
     'AWX_PERF_STATS_ENABLED',
     field_class=fields.BooleanField,
-    default=True,
+    default=False,
     label=_('Enable Performance Stats'),
-    help_text=_('Collect resource profiling data for playbook runs'),
+    help_text=_('Collect resource profiling data for playbook runs.  Requires Ansible 2.8 or greater.'),
```

Comment:
I think we should add additional text here stating that this feature will affect job run performance and that it should be toggled off once the desired profiling data is collected. Tagging @tvo318 here for proper wording.